pkgs/development/libraries/libroxml/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "libroxml-2.3.0";
+  src = fetchurl {
+    url = "http://download.libroxml.net/pool/v2.x/libroxml-2.3.0.tar.gz";
```

Comment:
For the last part of the URL you can just do `${name}.tar.gz` if you'd like (for example to have slightly less things to change if you ever update it). You will need to do `stdenv.mkDerivation rec {` for that though.